pyproject.toml

```diff
@@ -1,59 +1,63 @@
-[tool.poetry]
-package-mode = false
+[project]
 name = "romm"
-version = "0.0.1"
 description = "A beautiful, powerful, self-hosted rom manager"
 license = "GNU AGPLv3"
-repository = "https://github.com/rommapp/romm"
-authors = ["Zurdi <zurdi@romm.app>", "Arcane <arcane@romm.app>"]
+readme = "README.md"
+authors = [
+  { name = "Zurdi", email = "zurdi@romm.app" },
+  { name = "Arcane", email = "arcane@romm.app" },
+]
+requires-python = "^3.12"
+dependencies = [
```

Comment:
should not go under `[project.dependencies]`?

Reply:
Yes, but as it's just an array of strings, that's the way to represent it in TOML.
Example: https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#dependencies-optional-dependencies